package.json

```diff
@@ -44,6 +44,7 @@
     "ts-helpers": "^1.1.1"
   },
   "dependencies": {
+    "ajv": "^4.11.3",
```

Comment:
Also for this warning you can check here: https://github.com/webpack/webpack/issues/196